deploy/action.yml

```diff
@@ -151,6 +155,7 @@ runs:
           --environment '${{ inputs.environment }}' \
           --workload-type '${{ inputs.workload-type }}' \
           --runtime-cloud-provider '${{ inputs.runtime-cloud-provider }}' \
+          --helm-chart-repository-url '${{ inputs.helm-chart-repository-url }}' \
```

Comment:
den burde settes med env siden den kan være tom, se kommentar over listen med `3LV_`

Reply:
må legge til `EnvVars` her isåfall https://github.com/3lvia/cli/blob/trunk/pkg/deploy/deploy.go#L147-L150